torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -4818,14 +4896,13 @@ def _test_post_localSGD_optimizer_parity(self, averager, grad_is_view):
                 loss = loss_fn(output, target)
                 loss.backward()
                 opt.step()
-                averager.average_parameters(net.parameters())
+                averager.average_parameters(opt.param_groups)
```

Comment:
Let's only support parameters instead of parameter groups here. We can add a comment that all the parameters in the model do not satisfy `grad = None`, so all the parameters need to be averaged.

Reply:
fix